string type
string parameters
---
bool success
string message
string id
